src/assets/css/style.css

```diff
@@ -0,0 +1,166 @@
+:root {
+  --my-main-black: rgb(0, 0, 0);
+  --my-golden-orange: rgb(240, 138, 75);
+  --my-sky-blue: skyblue;
+  --my-header-footer-bg-color: rgb(243, 241, 238);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 16px;
+  font-family: 'Lato', sans-serif;
+}
+
+/*
+  Buttons
+*/
+
+/* buttons */
+button {
+  font-family: 'Lato', sans-serif;
+  border: none;
+  cursor: pointer;
+  height: 1.5rem;
+  box-shadow: rgba(119, 117, 117, 0.66) 0 1px 4px;
+  background-color: #fff;
+  padding: 1%;
+}
+
+.add-button {
+  border: none;
+  height: 2rem;
+  width: 4.5rem;
+  font-family: 'Lato', sans-serif;
+  box-shadow: rgba(0, 0, 0, 0.66) 0 1px 4px;
+  cursor: pointer;
+  align-self: end;
+}
+
+.add-button:hover,
+button:hover {
+  background-color: rgb(209, 200, 190);
+}
+
+/*
+  Header
+*/
+.header-container {
+  box-shadow: 4px 3px 2px;
+  background-color: rgb(243, 241, 238);
+  width: 100%;
+  height: 90px;
+}
+
+nav ul {
+  display: flex;
+  gap: 60px;
+  align-items: center;
+  margin-left: 20%;
+  font-weight: bolder;
+  padding-top: 20px;
+}
+
+ul li {
+  list-style: none;
+}
+
+ul a {
+  text-decoration: none;
+  color: black;
+}
+
+.logo {
+  border: 2px solid;
+  padding: 10px;
+  font-weight: bold;
+  border-radius: 50%;
+}
+
+nav a:hover {
+  text-decoration: underline;
+}
+
+/*
+  Main page
+*/
+main {
+  height: 70vh;
+  margin-top: 90px;
+}
+
+.main-items {
+  width: 100vw;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr); /* 3 columns */
+  grid-auto-rows: minmax(0, 1fr); /* Allow rows to be repeated */
+  gap: 1%;
+  margin: 0;
+}
+
+.item {
+  /* flex: 1; */
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid var(--image-placeholder-color);
+  margin: 0 4%;
+  position: relative;
+}
+
+.item-image {
+  display: block;
+  height: 80%;
+}
+
+img {
+  width: auto;
+  height: auto;
+}
+
+.item-wrapped-elements {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* Footer section */
+
+.footer-section {
+  border-top: 1.5px solid black;
+  border-radius: 0.1%;
+  width: 100%;
+  height: 5%;
+  text-align: center;
+  background-color: var(--my-header-footer-bg-color);
+  position: fixed;
```

Comment:
Kindly remove the positioning on the footer because in its fixed position it blocks some useful information on the web page.